src/me/philipsnostrum/bungeepexbridge/modules/PermPlayer.java

```diff
@@ -34,8 +35,9 @@ public PermPlayer (UUID uuid){
 
     public static PermPlayer getPlayer(UUID uuid){
         for(PermPlayer permPlayer : getPermPlayers())
-            if(permPlayer.getUuid().toString().equalsIgnoreCase(uuid.toString()))
+            if(permPlayer.getUuid().toString().replace("-","").equalsIgnoreCase(uuid.toString().replace("-",""))) {
```

Comment:
zPermissions stores UUID's trimmed. So I just converted the getPlayer to that, should still work with pex and sexypex.